scripts/release-notes.mjs

```diff
@@ -0,0 +1,59 @@
+import fetch from 'node-fetch';
+import querystring from 'querystring';
+
+const ptToken = process.env.PT_TOKEN;
+
+const features = [];
+const bugs = [];
+
+
+async function collectStories(projectId, search) {
+  const urlQuery = querystring.stringify(
+    {
+      query: search
+    }
+  );
+
+  // Documentation of this API is here: https://www.pivotaltracker.com/help/api/rest/v5#Search
+  const url = `https://www.pivotaltracker.com/services/v5/projects/${projectId}/search?${urlQuery}`
+
+  const response = await fetch(url, {
+    headers: { 'X-TrackerToken': ptToken }
+  });
+  const json = await response.json();
+  const stories = json.stories.stories
+  for(const story of stories) {
+    if(story.story_type === "feature"){
+      features.push(story);
+    }
+    if(story.story_type === "bug"){
+      bugs.push(story);
+    }
+  }
+}
+
+const ptLabel = process.argv[2];
+const search = `label:${ptLabel} includedone:true`;
+// Orange
+await collectStories(2441249, search)
+// Teal
+await collectStories(2441242, search)
```

Comment:
Named constants would be more self-explanatory here, e.g. `tealProjectId`.

Reply:
Good point